### ISSUE TITLE
[BUG]: annotation `ignore-check.kube-linter.io/access-to-secrets` does not work

### ISSUE THREAD

**Issue by kyubisation** [stackrox/kube-linter#605]
**System info:**
 - OS: Linux, Ubuntu 22.04.1 LTS

**Describe the bug**
kube-linter allows ignoring specific violations via annotation: https://docs.kubelinter.io/#/configuring-kubelinter?id=ignoring-violations-for-specific-cases
This does not work for `access-to-secrets` in a role binding.

```
.helm-template-output.yaml: (object: <no namespace>/argocd-image-updater rbac.authorization.k8s.io/v1, Kind=RoleBinding) binding to "argocd-image-updater" role that has [get list watch] access to [secrets] (check: access-to-secrets, remediation: Where possible, remove get, list and watch access to secret objects in the cluster.)
```

**To Reproduce**

- Apply `ignore-check.kube-linter.io/access-to-secrets`
- `kube-linter lint .helm-template-output.yaml`

**Sample YAML input**

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: argocd-image-updater
rules:
- apiGroups:
  - ""
  resources:
  - secrets
  - configmaps
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - argoproj.io
  resources:
  - applications
  verbs:
  - get
  - list
  - update
  - patch
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - create
---
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: argocd-image-updater
  annotation:
    ignore-check.kube-linter.io/access-to-secrets: "Argo CD Image Updater needs access to secrets in the Argo CD namespace/project"
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: argocd-image-updater
subjects:
- kind: ServiceAccount
  name: argocd-image-updater
```

**Expected behavior**
The error should be suppressed.

**Screenshots**

-

**Additional context**

- Thank you for providing this tool!

**Comment by kyubisation** [comment]
Unfortunately I'm apparently just unable to read and missed the s in `annotation(s)`.
Sorry about that.
